tests/cross/src/test/scala/tests/pc/CompletionScaladocSuite.scala

```diff
@@ -0,0 +1,194 @@
+package tests.pc
+
+import tests.BaseCompletionSuite
+
+object CompletionScaladocSuite extends BaseCompletionSuite {
+  check(
+    "scaladoc-methoddef",
+    """
+      |object A {
+      |  /**@@
+      |  def test(x: Int, y: Int): Int = ???
+      |}""".stripMargin,
+    """|/** */Scaladoc Comment
+       |""".stripMargin
+  )
+
+  check(
+    "scaladoc-classdef",
+    """
+      |object A {
+      |  /**@@
+      |  case class(x: Int) {}
+      |}""".stripMargin,
+    """|/** */Scaladoc Comment
+       |""".stripMargin
+  )
+
+  check(
+    "scaladoc-no-completion",
+    """
+      |object A {
+      |  /**@@
+      |}""".stripMargin,
+    ""
+  )
+
+  checkEdit(
+    "scaladoc-methoddef-edit",
+    """|object A {
+       |  /**@@
+       |  def test1(param1: Int, param2: Int): Int = ???
+       |  def test2(param1: Int, param2: Int, param3: Int): Int = ???
+       |}
+       |""".stripMargin,
+    """|object A {
+       |  /**
+       |  *
```

Comment:
shouldn't the `*` be aligned with the first `*` in `/**` (following Scaladoc style)?

Reply:
nevermind, see my comments below